src/tabs/ConstantsPage.js

```diff
@@ -38,21 +38,19 @@ import { hex_to_ascii, int_to_ascii, int_to_hex } from '../utils/VariableFormat'
 import { constants_default_values } from '../utils/DefaultValues';
 
 export const ConstantsPage = props => {
-    const USE_ROBONITOR = true;
+    const USE_ROBONITOR_PROTOCOL = true;
 
     const [constantList, setConstantList] = useState(constants_default_values);
 
     useEffect(() => {
         async function getConstant() {
-            const constant = await GetConstantsLabels();
+            let constant = await GetConstantsLabels();
             constant = constant == null ? constants_default_values : constant;
```

Comment:
Isso aqui pode ser assim, esse operador `??` é bem útil em várias situações
`const constant = (await GetConstantsLabels()) ?? constants_default_values;`

Reply:
mas se GetConstantsLabels() não for NULL "constant" recebe  o que?

Reply:
O valor retornado pela função
`a ?? b` é basicamente o mesmo que `a != null ? a : b`